### DISCUSSION TITLE
Upward continuation of multiple flight lines

### DISCUSSION THREAD

**Discussion by lawrenceabird** [fatiando/community#120] [Q&A] [answered]
Hi folks:

I am using Equivalent Sources to upward continue free-air gravity data to a consistent altitude. I have a number of different flight lines, all flow at different altitudes, and data have had a ~150s filter applied along each flight line. Given the filtering, I am wondering if it is best/appropriate to fit equivalent sources and upward continue _each individual line_ separately, or to deal with all of the data at once and fit an Equivalent Source model to the entire dataset at once?

Any advice appreciated here!

Thanks,

Lawrence

**Comment by mdtanker** [comment, marked as answer]
Hi @lawrenceabird, we had a little discussion on this here: https://github.com/orgs/fatiando/discussions/70 but we never came to a solid conclusion.

Is your data already levelled? If not, I think it makes most since to individually upward continue the lines to the same elevation so that the crossover errors can be correctly calculated. Then use these individually upward continued lines to level the dataset.

Interested to hear what others have to say about this as well!

**Comment by leouieda** [comment]
I mostly agree with @mdtanker on this. Crossover analysis on data at different heights never made any sense to me. The main difficulty would be getting a good equivalent source model from a single line. I've never tried this but I fear you may lose some long wavelength signal because of the lack of sources to the sides of the lines. Some sort of remove-computer-restore procedure can help if this happens, for example removing a spherical harmonic model first and doing the upward continuation of the residuals then restoring the model at the new height.

Honestly, this is a pretty good research question. Please share what you come up with and any issues you face. I'd be very keen to see what happens 🤓
